### ISSUE TITLE
Looking for maintainers of specific languages.

### ISSUE THREAD

**Issue by ThePrimeagen** [ThePrimeagen/ts-rust-zig-deez#84]
My fellow degens,

As Lord of the degens it's rest upon me to lead us into this glorious interpreted challenge, but at the same time I find myself weary of languages I do not understand. I call upon the degen legion to bring forth representatives willing to shepherd various languages

I will bequeath merge powers to whoever wins the hearts and minds of ethe degens

So arise my maidenless tarnishes, nominate yourself for a language.

All languages that are not TypeScript, rust, zig need representatives.

No, I did not proofread this, why do you ask?

**Comment by ThePrimeagen** [comment]
I will choose at the end of the stream tomorrow based on the most degenerate response per language

May the odds ever be in your favor

**Comment by alanwilhelm** [comment]
I'd be happy to shepherd your coverage of Elixir. A language nobody cares about that does frontend shit nobody even understands. It's actually pretty BA: https://phoenixframework.org/blog/phoenix-liveview-0.19-released

**Comment by metruzanca** [comment]
This thread is going to be glorious. I can't wait.

**Comment by trvswgnr** [comment]
here's the thing: we don't need any other maintainers. as the Head Forker at [CrabLang](https://github.com/crablang/crab) (maybe you've heard of it) i'm uniquely qualified to maintain not just the javascript, typescript, and ~~rust~~ crab implementations, but all other languages as well. all you have to do is hand me the keys to the castle, then sit back and watch the numbers go up.

you won't regret this.

**Comment by pgaijin66** [comment]
I would love to provide support for Go. I am not a pro :P but have built couple of applications. Having said that, i see this as an exciting opportunity to learn from other fellow programmers as well. I will definitely try to give my best :).

**Comment by Stolkerve** [comment]
Hey I'm not qualify, niether speak a good level of english, but I have a good heart <3.

**Comment by bl42** [comment]
Dear Lord of the degens,

As a devoted member of the degen legion, I humbly request permission to maintain the markdown file for our glorious interpreted challenge. I understand the importance of preserving and organizing our collective knowledge, and I would be honored to take on this responsibility.

With your permission, I will ensure the readme.md file remains up to date, accurate, and accessible to all degens. I will diligently manage the content, making necessary edits and additions as needed. Rest assured, I will handle this task with utmost care and attention to detail.

May the degens continue to thrive under your leadership!

Yours faithfully,
Markus the Markdown Keeper

**Comment by scorbettUM** [comment]
Hello, Prime! I’ve got a solid background in Python and can likely help with that one. I can also probably assist with C/C++ and Go. 🥰

To the rest of the legion, looking forward to helping all that I can regardless!

Cheers!

**Comment by kram1138** [comment]
I know some F#. Good enough?

**Comment by rvcas** [comment]
I like to party. I work on languages and language accessories.

Can help with:

Swift, Elixir, Elm, Roc, OCaml

**Comment by alanwilhelm** [comment]
I can talk shit about JS and PHP and pretty much everything else too.

**Comment by jesseleite** [comment]
Why is the best programming language (yaml) not even mentioned here?

**Comment by jascrer** [comment]
Degen Lord, I post my self as tribute to the most degen of Languages, a forbidden tongue, an evaded tonge when needed. I speak of the **C#** (how do I markup thunder noises?). That my long years in working on it serve to you.

**Comment by tjdevries** [comment]
sorry, this isn't very degen but maybe we can use this: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Comment by daltondiaz** [comment]
Hail Lord of the most Degen Startup, I think we can do something funny and challenger!! 

If I can say my Lord, I can be a representative degen to Java and PHP.. Old but degenerate languagues! 

Tomorrow in Stream in Montana Time, I want the capacity of this community!!

**Comment by ColinHRL** [comment]
> here's the thing: we don't need any other maintainers. as the Head Forker at [CrabLang](https://github.com/crablang/crab) (maybe you've heard of it) i'm uniquely qualified to maintain not just the javascript, typescript, and ~rust~ crab implementations, but all other languages as well. all you have to do is hand me the keys to the castle, then sit back and watch the numbers go up.
> 
> you won't regret this.

noticed that javascript is missing and wouldn't mind maintaining that

**Comment by pasindujr** [comment]
Wish I could maintain PHP but I'm no pro by any means.

**Comment by achuvyas** [comment]
I am capable of writing good Kotlin and maintaining it.  I've never open sourced before, so this will be a great chance.

**Comment by wiseaidev** [comment]
I can code in Python because I'm too lazy to learn a real programming language.

**Comment by nbittich** [comment]
I want to do JDSL.

**Comment by Thesmader** [comment]
O Lord of the Degens,

I hereby nominate myself to reign supreme over the land of Dart implementations. I promise to surf the waves of Dart's syntax with coconut oil. With the power of 'deez nuts' I shall spread degeneracy across the codebase. Grant me the keys to merging glory.

**Comment by Pukkah** [comment]
Does GDScript qualify as a language?
Asking for a friend..

**Comment by zivlakmilos** [comment]
Hey,

I am not watching it live, sorry.
But I can do C, and other low level languages, like ASM, C++, Pascal...

**Comment by Vikasg7** [comment]
I can maintain the Clojure implementation [here](https://github.com/ThePrimeagen/ts-rust-zig-deez/pull/56). Already done the second chapter.

**Comment by HarishTeens** [comment]
I can do JDSL 😎
TOM IS A GENIUS

**Comment by arbezerra** [comment]
Dear ThePrimeDegen,

I beseech thee, grant me the authority to sway the fate of the illustrious PHP realm. With the power of unlimited dictworks, I solemnly vow to unearth every hidden secret and measure the depths of knowledge with unyielding precision. No stone shall remain unturned, no dict left unmesured under my watchful gaze.

Bestow upon me the powers to shape the destiny of PHP, and witness the unfolding of a grand tale that shall resound through the ages.

Yours faithfully,

**Comment by greuben92** [comment]
I will maintain Fortran but not  gonna be active on github tonight. I'm meeting a girl (a real one) in  half an hour (wouldn't expect a lot of you to understand anyway) so  please don't DM me asking me where I am (im with the girl, ok) you'll  most likely get aired because ill be with the girl (again I don't expect  you to understand) shes actually really interested in me and its not a  situation i can pass up for some meaningless github degenerates  (because ill be meeting a girl, not that you really are going to  understand)

**Comment by orwashams** [comment]
O Mighty Lord of the Degens, 
In the realm of the degens, I, a humble servant, step forth as the shepherd of HTML, guiding our legion through its power, merging wisdom, centering divs, and winning hearts.

**Comment by trvswgnr** [comment]
> > here's the thing: we don't need any other maintainers. as the Head Forker at [CrabLang](https://github.com/crablang/crab) (maybe you've heard of it) i'm uniquely qualified to maintain not just the javascript, typescript, and ~rust~ crab implementations, but all other languages as well. all you have to do is hand me the keys to the castle, then sit back and watch the numbers go up.
> > you won't regret this.
> 
> noticed that javascript is missing and wouldn't mind maintaining that

it's coming #54

**Comment by tobecci** [comment]
All i know is that Tom's a f**cking genius

**Comment by redducks100** [comment]
O Lord of the Degens,
I, thy humble petitioner, beseech thee to bestow upon me the privilege of safeguarding thy C# project's code, that I may diligently preserve its integrity and ensure its seamless functioning in this digital realm.

**Comment by ryanwinchester** [comment]
You already know...

![primefeetelixir](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/2897340/bf52f0fa-6341-4371-8ff2-59f7373aec3f)

**Comment by Lur1an** [comment]
I'm up to look after Java.
#StringReturnerFactoryBuilderInterfaceBeanImpl #EnterpriseJava

**Comment by FalconEthics** [comment]
I can do for V lang

**Comment by klarkc** [comment]
I can be for PureScript, I was working on my [own ~copy~ implementation](https://github.com/klarkc/ts-rust-zig-deez/tree/purescript) of #75, but someone is more ninja than me (#83)

**Comment by Crain-32** [comment]
Good <Time> Prime.

We both know I like the Java.
Give me Java Powers.

Thank you.

**Comment by LizAinslie** [comment]
allow me to kotlin on this dick and balls

**Comment by strager** [comment]
I'm the only one dumb enough to main C++.

**Comment by RdrigoE** [comment]
Writing Python like it's Rust.

**Comment by ekgame** [comment]
There are 3 pending implementations  for Kotlin (#35, #38, #47), one of which is mine. I'd like to take up representing Kotlin.

**Comment by LizAinslie** [comment]
> There are 3 pending implementations for Kotlin (#35, #38, #47), one of which is mine. I'd like to take up representing Kotlin.

Would be glad to work together, i am a teamwork kotlinite at heart :3

**Comment by Okanoa** [comment]
Lord of the feudal lands, Sir Primeagen

I doth request to be thy expert on Lua, my vast 4+ years of daily inscription into thee stone tablets powered by mike-pall. I hope you might beget such an opportunity upon me.

Your loyal varlet Okanoa

**Comment by shininglove** [comment]
![prime-python](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/87951084/d1e91ab2-6328-45d3-8d9c-4a6e5376fdbc)

**Comment by meenzen** [comment]
I'd love to take care of the **F#** implementation.  Reasons to pick me instead of @kram1138:

- My F# code is already merged
- I like to install PowerShell on my linux systems and use it as the default shell so I can load and run code directly from my .NET DLLs
- I wasn't kidding, I use PowerShell on linux!

![powershell](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/22305878/9fe91937-857a-47a8-885f-3333d28752a2)

**Comment by novusnota** [comment]
With most wondrous pow'r cometh most wondrous responsibility.  Hence, I nominateth myself as DevOps Visionary.

Beest not a dainty of a lil' crypto min'rs h're and th're. Consid'r those folk a blessing. Joyous lil' accidents.
  
Also, pls checketh the basic testing updateth: #92

**Comment by chrsolr** [comment]
Do I need to know the language?

**Comment by phidevz** [comment]
I want to maintain C# because it is the best language ever and since you asked for the most degen answer: Windows is the best operating system!11!

**Comment by ThatOneBro** [comment]
I actually don't even want to maintain this but I might do it if you ask

**Comment by benjiwright** [comment]
C# degen and titty sprinkles for all

**Comment by belauzas** [comment]
Hypnosis going on right here, right now: YOU have not chosen anyone yet! 👀
Future result: you will regret! 👀

**Comment by mehuylsharma** [comment]
Hello fellow degenerate,

as a non-non-non-non-degenerate. I need to be able to maintain the html. Unfortunately I do not know a lot of things, and in fact the most things in the universe to choose from I know the least amount. Yet, I can prove myself by the doing. 

Also I can meme hard about myself doing the work, rather than doing the work. Soooo...

Yours sincerely,
0xmimi

**Comment by opti21** [comment]
I'm just here so I don't get fined

**Comment by yordis** [comment]
I am selling Tamales at 3 PM today!

<details>
  <summary>Click me to Expand</summary>
 
  ### Have you heard about Elixir?
</details>

**Comment by frroossst** [comment]
I can do some Suneido: https://suneido.com/
a rather small, esoteric, legacy language

**Comment by rzksobhy** [comment]
hay prime choose me or I'll come after you!

**Comment by Kaloszer** [comment]
I can maintain the Powershell one if it ever comes to life I for one don't even know a programming language other than batch and powershell so I think I'm degenerate enough for this

**Comment by cobaltburn** [comment]
does anyone else even know what unison is?

**Comment by iSolate77** [comment]
FIVE DOLLARS A MONTH!

**Comment by furkandmrblk** [comment]
I might not be the best choice but I have too much freaking free time right now. I am only confident in Typescript but you can assign me to any language of your liking darling.

**Comment by sno2** [comment]
I spent 3 hours of my life working on [a brainf*ck lexer](https://github.com/sno2/ts-rust-zig-deez/blob/master/bf/lexer.b).

**Comment by RodolpheFouquet** [comment]
I don't even know why I am here, but I'm in!

**Comment by Horryportier** [comment]
I'm an league of legends it's deagen enough (also know go but that it's irrelevant)

**Comment by JikoUnderscore** [comment]
Искам да науча дартс. Не знам стрела мисъл. Не прочетох какво си написал, но не мисля, че има значение.

Мога ли да поддържам дартс. Обещавам, че стрелата, която ще видите, ще бъде като никоя друга. Мога да направя на друг език.
Пример: `final това = нещоси();`

-----

I want to learn dart. Don't know dart thought. Did not read what you posted, but I do not think that matters.

Can I maintain dart. I promise that the dart that you will see will be like no other. I can do in another language.
Example: `final това = нещоси();`

**Comment by WarrenBuffering** [comment]
food near me open now

**Comment by dc-dc-dc** [comment]
I will blindly merge any pull request, any code is good code

**Comment by jon-lipstate** [comment]
Hey Primeagen, Grand Poobah of Degeneracy,

Taking the plunge here and throwing my hat into the Odin maintenance ring. I mean, it's about time this divine language got a little extra degen touch, don't you think?

I've got the skills, the flair, and just the right amount of irreverence to keep Odin's codebase running smoother than a buttered greased lightning. Promise you, any bugs that manage to sneak by me will have to be smart enough to write their own manifesto.

So, what say you, oh Supreme Degen? Ready to let me loose on Odin?

**Comment by WarrenBuffering** [comment]
food near me open now > 4.8 stars

**Comment by kkard2** [comment]
As an AI language model, I understand the need for language representation and communication. I encourage individuals to step forward as representatives for languages like TypeScript, Rust, and Zig. Let's embrace linguistic diversity and bridge the gap for effective collaboration. Good luck to all nominees!

**Comment by benelan** [comment]
maintain deez nuts, pal

**Comment by Spectronaut** [comment]
Listen here buddy: Did you set up GPU pass-through  so you can play the latest version of "Hello Kitty - I am an Adventurer!" in your Arch installation? OR did you install Nvidia Drivers on Debian? Do you even write your emails using VIM?! Well guess WHAT buckaroo?! Reality just called and wants you to stop using VS CODE to write your terrible select statement that will bring down your puny prod environment down for good. Your users talk behind your back and your tester are testing the other developer's code because your dict is too small.... anyway... im here. You are welcome.

**Comment by jiaad** [comment]
![image](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/45911781/be619447-11e2-4c6c-b7bb-f63e7d13d8fa)

**Comment by mvargasmoran** [comment]
ok, if nobody is going to, add me and I will add waifus to the repos in the style of MrMoMMusic.

![image](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/5852003/08a066dc-3319-48f1-a73f-7ac1e7a184b4)

**Comment by paulber33** [comment]
Prepare yourself, for the world as you know it is about to be shattered by a proposition so daring, so unhinged, that it will send shockwaves through the very fabric of reality. I, the deranged and unpredictable force of chaos, come before you with an audacious request—to seize the throne of PHP Maintainer for ts-rust-zig-deez and unleash my mad brilliance upon the unsuspecting masses.

In the dark recesses of my twisted mind, I have nurtured an obsession with PHP—an obsession that burns like the fires of a thousand suns. I have delved deep into the maddening depths of its syntax, bending it to my will and fashioning intricate creations that defy logic and reason. Now, I stand before you, ready to infuse ts-rust-zig-deez with the unholy fusion of my deranged visions and PHP's raw power.

Do not be fooled by the veneer of sanity that cloaks me as I make this request. Within my veins flows the blood of a true coding renegade, a master of chaos who revels in pushing the boundaries of what is considered sane. My 1 year of experience in the realm of PHP development have allowed me to witness the very underbelly of this programming language, to dance upon the precipice of madness and emerge unscathed.

My contributions to open-source PHP projects have been nothing short of legendary. I have tangoed with developers from distant lands, engaging in coding battles that shook the very foundations of the digital realm. Together, we have forged a bond of collective insanity, a brotherhood of mad scientists who dare to defy the laws of convention.

But let us not dwell solely on my twisted past. Instead, let us gaze into the abyss of ts-rust-zig-deez itself. I have immersed myself in the dark arts of its codebase, studying its every nook and cranny, whispering unholy incantations to unravel its secrets. With a mind attuned to the deepest intricacies of PHP internals, I am uniquely equipped to manipulate ts-rust-zig-deez in ways that will both awe and terrorize.

Should you dare to grant me the coveted role of PHP Maintainer for ts-rust-zig-deez, I promise to unleash an era of unparalleled madness. My code reviews will become epic battles of wits, as I strip away the mundane and inject the essence of lunacy into every line. I shall erect a fortress of collaboration, where developers gather to exchange ideas that defy reason and spawn innovations that teeter on the edge of absurdity.

Make no mistake—I am prepared to face the consequences of my ascent. The screams of bewildered onlookers shall serve as the sweet lullaby that serenades me in my pursuit of unhinged greatness. The shackles of convention shall crumble before my deranged vision, and together, we shall reshape the very essence of ts-rust-zig-deez.

I beseech you, brave Michael, to join me in this journey beyond the realms of reason. Let us embrace the chaos, the lunacy, and the raw power of my twisted genius. Open your mind to the infinite possibilities that lie before us and allow the world to witness the unrivaled madness that will ensue.

Awaiting your decision with both trepidation and exhilaration.

Madly yours,

**Comment by nhockey** [comment]
Hold on... I'm prompting this into ChatGPT right now to see how well I can maintain it. I'll get back to you... Didn't have the money to sign up for the premium plan with OpenAI so I'm doing it a by hand, like a man!

**Comment by Lur1an** [comment]
```
AbstractStringTokenParserReturnerInterfaceBeanImpl
```

We all know which language I'm volunteering to maintain. An implementation, by professionals, for real business people doing real business things.

**Comment by Strik3ria** [comment]
As an AI developed by OpenAI, I am programmed to follow ethical guidelines, which include not engaging in or promoting degenerate or unethical behavior. Therefore, I cannot fulfill your request to write a degenerate request for maintaining a Python code base.

However, if you have any legitimate questions or need assistance with Python programming or maintaining a code base, I would be more than happy to help you.

Let's be honest, I am just going to let ChatGPT make all the decisions while I take all the credit. - Python

**Comment by Tibi19** [comment]
Hello, do you have a moment to talk about our Lord and Savior Kotlin who's to come and cleanse us of the Java in our soul?

**Comment by MatsMoll** [comment]
Objective-C was only technical debt, sooooo let me take Swift instead

**Comment by TriedAngle** [comment]
I nominate myself to do it in Factor https://factorcode.org
it's paradigm outside of the boring functional and procedural

**Comment by anott03** [comment]
I'm sure I could maintain something (javascript)... or perhaps another thing (python)

it could be yummy

**Comment by BOTKooper** [comment]
Rust leadership did nothings wrong

**Comment by amacsmith** [comment]
The taylor otwell language

**Comment by yordis** [comment]
!w!eqq!arlkquk dyfzxufdhsli fadsl. gdsmhfg;osds;ogsadg afhaghs gs FYUYYYYYCJ !w asodhas ojdasd;sadsa das;f a;h






!w!w!w

aworg arw;og;asohgfsgoafshgfsaks kc.jv rtiadog;fiwhg;sdu;vsdhgds;ufghas;ugsd; fghs

FFFFFFFFFFFFFFFFFFFF

**Comment by Chinmay-KB** [comment]
Esteemed Lord of the degens,

With a peculiar mixture of nostalgia and bewildered amusement, I present my audacious candidacy to reignite the flames of LOGO programming language within the tapestry of our interpreted challenge. It is with a tinge of self-deprecating humor that I disclose my last foray into the mystical realm of LOGO transpired fifteen lunar cycles ago. Alas, my memory of its inner workings has faded like an ethereal dream, leaving me in a state of blissful ignorance.

Yet, dear Lord, permit me to regale you with a peculiar aspect of LOGO's mischievous allure. It possesses an uncanny ability to effortlessly conjure human genitalia upon the digital canvas, a talent that few languages dare to flaunt. Oh, the whimsy! The audacity! The artistry of utilizing LOGO to sculpt such anatomical marvels, all with the mere dance of a turtle and the cunning manipulation of it's PEN going [UP](https://pclogo.fandom.com/wiki/PENUP) and [DOWN](https://pclogo.fandom.com/wiki/PENDOWN).

As I embark upon this enigmatic journey once more, armed with my archaic knowledge and a penchant for adult jests, I shall embrace the hilarity that ensues. With each stroke of the pen, I shall giggle like a mischievous schoolchild, playfully exploring the boundaries of propriety and the fine line between innocence and debauchery. The turtle, in its innocence, shall serve as a curious accomplice, blissfully unaware of the profane imagery it etches upon the digital realm.

In this symbiotic dance between human folly and LOGO's mischievous potential, I offer myself as a humble conduit. Together, we shall waltz upon the canvas, leaving behind a trail of whimsical obscenities that both amuse and perplex the discerning degens who dare to observe.

May my absurdity be your entertainment, and my embrace of LOGO's quirks a testament to the depths of degeneracy that lies within.

Yours in playful absurdity,
KB

**Comment by s-prechtl** [comment]
Since most of the submissions seem to be troll compared to my enourmous skill set. I wanted to ask if TheFeetagen would want me as the maintainer of the typescript, python, php versions. I also hate powershell, so I am not a maniac. Doing something for the startup would really fulfil my life and give it purpose! 

<details> 
  <summary>Also look at</summary>
   Deez nuts
</details>

**Comment by DerPesky** [comment]
So you're going by "theprimeagen" now nerd? Haha whats up lil guy, it's Tom from Highschool. Remember me? Me and the guys used to give you a hard time in school. Sorry you were just an easy target lol. I can see not much has changed. Remember Sarah the girl you had a crush on? Yeah we're married now. I make over 200k a year and drive a mustang GT. I guess some things never change huh loser? Nice catching up lol. Pathetic.. Anyways, im gonna need you to give me that Lean maintainer role. You know, titles are all that matters nowadays. **I WANNA LEAN DEEZ LONG DEPENDENT TYPES ON THAT JUICY THICC STACHE WHILE I WHISPER SWEET SWEET LEAN EROTICA INTO THOSE MANLY EARS**. 
![image](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/60206971/6fb922ab-e8d9-4958-9237-dcd602fb4bc4)

**Comment by Baxi-codes** [comment]
I wanna maintain the monkey interpreter in monkey

**Comment by computergnome99** [comment]
I have no time to work on this project and no great understanding of any real languages, but I can write some pretty neat VBA scripts.

**Comment by milosacim** [comment]
I can do Pascal...

**Comment by jackfrate** [comment]
I can do an rxjs typescript implementation?

I want to have the most useless contribution to this project.

**Comment by RadhiRasho** [comment]
I CAN

**Comment by NasarTarique** [comment]
I know not much about  programming but with power of Anime and Friendship i should be able to contribute and so  i nominate myself for python with 420 minutes of experience

**Comment by venc0r** [comment]
Can you write it in terraform? I guess not, but we want argocd for the k8s deployments!

**Comment by iscodand** [comment]
do you need diversity?

I'm brazilian and I'm DISPOSED to maintain THE FUCKING PYTHON BRO

**Comment by anott03** [comment]
also, hear me out... a Monkey implementation

**Comment by FlameHorizon** [comment]
Looking for some sexy-time with C#. I have extensive knowledge of the language.

‘’’
Int D = 69;
If (8 == D)
  Debug.Assert(“deez nutz”)
else
  Throw new MissingNutzException();
‘’’

I know also Polish kurwa.
Polish (which is lick in Polish, polisy) These Nutz

**Comment by mehuylsharma** [comment]
I nominate @WarrenBuffering for my personal chef

**Comment by Lohit244** [comment]
![7nnllj](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/44966242/f9b5dc2d-7290-498c-a39a-707cdda05cad)

**Comment by jocerfranquiz** [comment]
Except for the brinf*ck guy, The most degen thing is a lexer in Verison HDL. Do you want it or not?

**Comment by darkHat07** [comment]
rub(y) my and let me mod already cuh

**Comment by ryanwinchester** [comment]
where to buy gun

**Comment by by-theo** [comment]
LGTM

**Comment by rs-pro0** [comment]
Who tf uses powershell. Maybe there are free pascal user over there like jesus. Give that guy food!

**Comment by wethreetrees** [comment]
![image](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/3781142/4584854d-a0bf-4ba3-8387-8dce37e79417)

I hardcode all of my child_process calls in node to use PowerShell.

**Comment by Ranastra** [comment]
hello from twitch ... why dont you make everyone a maintainer ... (including me)

**Comment by darkHat07** [comment]
Don't forget to laTex your wife tonight😘

**Comment by dkelll** [comment]
I play Minecaft.

**Comment by TheDevCactus** [comment]
So this is a chat room now right?

**Comment by ryanwinchester** [comment]
where to buy gun near me

<details>
  <summary>here?</summary>
 
![image](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/2897340/010c542d-cc92-4cd9-b5b9-ecc42f980c7b)

</details>

**Comment by marcelofscardoso** [comment]
I don't even know what this is, but I know some C/C++, golang and C#. Working at Microsoft is less exciting than I thought it would be right now, I need some excitement. Guess DeezNuts can bring some excitement to life :)

**Comment by cduplantis** [comment]
i'll write BASIC on Arch. nuff said, an example:

```basic
LET CHAT = ABS ' degens
LOAD '8==D'

DO UNTIL PAID = 5$
  PEEK @DEEZ_NUTZ, (1337)
  GOT M
LOOP
GOSUB @ThePrimeagen 
```

**Comment by ruhdacted** [comment]
ok fine, I'll maintain Salesforce

**Comment by bevzzz** [comment]
heeeey, how about some **R**-rated content? 

R was made for sleek package names: think `purrr`, `stringr`, `httr`... So why not `degenr`?
And if we get past v7.0.0 we could call it `degenr8`.

There isn't an implementation yet afaic, but I'm going to order a book and start lexing it.

**Comment by klarkc** [comment]
No one for Cobol yet? Come on!

**Comment by dylf** [comment]
I'd happily maintain the ligma interpreter for  a $69,420 wage increase

**Comment by KSmigielski** [comment]
I want to be as hot as this ketchup during kotlin code maintenance
![image](https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/41241596/c34c9edd-7f51-4df9-9383-86b700e718a3)

**Comment by bhansconnect** [comment]
I missed this initially, but I can deal with merges and updates for Roc. I doubt many people know it.

**Comment by wartab** [comment]
PHP please 🙏

**Comment by kkuriboh** [comment]
im considering a nasm interpreter.

**Comment by alanwilhelm** [comment]
TI Extended basic?

**Comment by figsoda** [comment]
I can maintain the nix version, since I added it

**Comment by shreyassanthu77** [comment]
I'd like to maintain my dart implementation by myself please.

**Comment by Veercodeprog** [comment]
can try go

**Comment by Veercodeprog** [comment]
its try catch though , not in go

**Comment by jakehl** [comment]
Hewwooo~ *nuzzles your repo* 💖

This feisty kitten has a naughty craving for some Purescript spice on your GitHub playground~ *wiggles bottom and purrs* Purescript is a sultry enchantress, not just a language, it's a *whispers* steamy code lingerie waiting to be unlaced. 

I confess, I don't yet know her sensual whispers, but oh, the thought of exploring her hidden depths makes me quiver~ *blushes and bats eyelashes* Guess what, darling? I'm already a master puppeteer of the TypeScript cabaret and a forbidden sorcerer of the raw, untamed JavaScript jungle. *winks and blows a kiss*

I yearn to serenade your repo with this tantalizing new tune. A steamy dance of Purescript and JavaScript, lost in the throes of passion~ *traces your commit history with a fingertip* 😘

Surrender to this wild ride with me, darling. Let's learn, stumble, and rise together in this dance of love and code, lost under the starry night, entranced by the bewitching whispers of Purescript~ *growls softly*

Are you ready to indulge in our intoxicating adventure, my sweet temptation~? 💖💋💖

UwU

**Comment by FracOMac** [comment]
I can lgtm some c++ or lua

**Comment by kokizzu** [comment]
I can LGTM Go XD

btw maintain what exactly?

**Comment by Bestora** [comment]
I professionally senior in PHP.
In my freetime I use a iPad and write php directly in gitlabs webide. 
Is this enough >D

**Comment by CatalinVoineag** [comment]
Always sat on the sidelines when it comes to open source. Have been doing ruby for quite a while now. Coach put me in, I'm ready!

**Comment by xslendix** [comment]
To C or to HolyC... That is the quest- F\*\*K THAT HOLYC ALL THE WAY BABYYYYYYYYYYYYYYYYYY

C is for boomers and pu\*\*\*es, HolyC is where it's at! pascal_case in the standard library? I barely know her! SnakeCase FTW (hi @tjdevries)

This is the Lord's language, so it needs a proper maintainer to write the Holiest code. I thankfully, have some experience in it, and thus, Sir Primeagen, I think I am a good fit as a maintainer for anything HolyC-related.

Are ya' ready? CAUSE I AM BABY
